x-pack/plugins/ml/common/types/data_frame_analytics.ts

```diff
@@ -56,16 +57,31 @@ export interface ClassificationAnalysis {
 }
 
 export type AnalysisConfig = estypes.MlDataframeAnalysisContainer;
+interface DataFrameAnalyticsMeta {
+  custom_urls: any[];
+  [key: string]: any;
+}
 export interface DataFrameAnalyticsConfig
   extends Omit<estypes.MlDataframeAnalyticsSummary, 'analyzed_fields'> {
   analyzed_fields?: estypes.MlDataframeAnalysisAnalyzedFields;
+  _meta?: DataFrameAnalyticsMeta;
 }
 
 export interface UpdateDataFrameAnalyticsConfig {
   allow_lazy_start?: string;
   description?: string;
   model_memory_limit?: string;
   max_num_threads?: number;
+  _meta?: Record<string, unknown>;
+}
+
+export function isDataFrameAnalyticsConfigs(arg: unknown): arg is DataFrameAnalyticsConfig {
+  return (
+    isPopulatedObject(arg) &&
```

Comment:
Nit: You could move part of the check to isPopulatedObject here like this:
```
isPopulatedObject(arg, ['dest', 'analysis', 'id']) && typeof arg.id === 'string'
```

Reply:
Updated in 7ee455c02c2252a159431305bf8b90cabaed6305